intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -227,6 +276,60 @@ def train(data):
 # From the above view, we can see that the runtime of ``enumerate(DataLoader)`` is reduced,
 # and the GPU utilization is increased.
 
+######################################################################
+# 6. Analyze performance with other advanced features
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# - Memory view
+# To profile memory, please add ``profile_memory=True`` in arguments of ``torch.profiler.profile``.
+#
+# Note: Because of the current non-optimized implementation of PyTorch profiler,
+# enabling ``profile_memory=True`` will cost about several minutes to finish.
+# To save time, you can try our existing examples firstly by running:
+#
+# ::
+#
+#     tensorboard --logdir=https://torchtbprofiler.blob.core.windows.net/torchtbprofiler/demo/memory_demo
+#
+# The profiler records all memory allocation/release events during profiling.
+# For every specific operator, the plugin aggregates all these memory events inside its life span.
+#
+# .. image:: ../../_static/img/profiler_memory_view.png
+#    :scale: 25 %
+#
+# The memory kind could be selected in "Device" selection box.
+# For example, "GPU0" means the following table only shows each operator’s memory usage on GPU 0, not including CPU or other GPUs.
+#
+# The "Size Increase" sums up all allocation bytes and minus all the memory release bytes.
+#
+# The "Allocation Size" sums up all allocation bytes without considering the memory release.
+#
+# - Distributed view
+# The plugin now supports distributed view on profiling DDP+NCCL.
+#
+# You can try it by using existing example on Azure:
+#
+# ::
+#
+#     tensorboard --logdir=https://torchtbprofiler.blob.core.windows.net/torchtbprofiler/demo/distributed_bert
+#
+# .. image:: ../../_static/img/profiler_distributed_view.png
+#    :scale: 25 %
+#
+# The "Computation/Communication Overview" shows computation/communication ratio and their overlapping degree.
+# User can get the load balance between workers from it.
+# For example, it the computation + overlapping time of one worker is much larger than others,
+# there may be a problem of load balance or this worker may be a straggler.
+#
+# The "Synchronizing/Communication Overview" shows the efficiency of communication.
+# "Data Transfer Time" is the time for actual data exchanging.
+# "Synchronizing Time" is the time for waiting and synchronizing with other workers.
+#
+# If one worker’s "Synchronizing Time" is much shorter than that of other workers’,
+# this worker may be a straggler which may have more computation workload than other workers’.
+#
+# The "Communication Operations Stats" summarizes the detailed statistics of all communication ops in each worker.
+
 ######################################################################
 # Learn More
 # ----------
```

Comment:
I like the call to action at the end. Great job! This article is great :)

Another call to action I would add is something like this: 
"feel free to open an issue here (hyperlink: https://github.com/pytorch/kineto/issues)

Reply:
@sabrina-smai Thank you very much! Your comments are very helpful! I've fixed all of them.